src/pullRequestInfo.ts

```diff
@@ -1,146 +0,0 @@
-import {callGitHubAPI} from "./utils/webServiceCall";
-import {Config} from "./utils/configFileReader";
-import async from "async";
```

Comment:
It may or may not be the case that after refactoring all the filtering by status to (synchronous) callers, you can eliminate this.